go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/jmespath/go-jmespath
+
+go 1.13
+
+require (
+	github.com/jmespath/go-jmespath v0.0.0-20180206201540-c2b33e8439af
```

Comment:
I think you should run `go mod tidy` in this repo, this line shouldn't be here IMO.

Reply:
yeah I saw it and fixed it. I usually don't edit those files. I must have `go mod init`ed with the wrong path which picked up the correct path as a dep.